### DISCUSSION TITLE
Speech to text module compatible with TTS numpy version

### DISCUSSION THREAD

**Discussion by xio28** [coqui-ai/TTS#2715] [General Q&A]
Hi, everyone,

A few days ago, I started using coqui TTS for my Flask API, and I have to say it's one of the best open-source TTS systems I've tried. It's quite easy to use, and the voices are some of the most realistic I've heard. The problem I'm facing is that I need two different libraries: one for text-to-speech (TTS) and another for speech-to-text (STT). The issue is that TTS uses an older version of numpy (1.21.6, while the latest version is 1.25.0). When I try to install modules like 'whisper' or any other, I encounter an error like this: 'ImportError: numpy.core.multiarray failed to import'. I deduced that it's due to the version mismatch because if I update numpy, 'whisper' works fine, but TTS gives me that error.

So, I wanted to ask if anyone knows of an STT module that is compatible with the version of numpy used by TTS. Well, if anyone is using STT module with coqui TTS in their python app, tell how to do it too, please?

Thanks in advance.

**Comment by shawnesquivel** [comment]
having te same issue
